news/_posts/2017-05-18-how-to-create-bug-report.md

```diff
@@ -0,0 +1,197 @@
+subtitle: "Three rules: up-to-date, reproducible, and detailed."
+date:   2017-05-18 09:10:11
+authors: [ xavierborderie ]
+icon: icon-bug
+tags:
+- contribution
+- forge
+---
+
+Every morning, the Product and QA team gather around a big screen, and review the new Forge tickets that were created since the day before. At the end of the meeting, all new tickets must be asserted, and if need be, placed into the hand of a team member.
+
+Reviewing Forge ticket can be very brain-consuming: there are times when we get only a quick sentence; other times when the report just gives the whole set of errors displayed by the browser; or sometimes all we get is a demand to fix something as soon as possible, with a phone number. It's not always easy to assert and respond quickly to such reports.
+
+On the other hand, we sometimes get tickets that are a pleasure to work with. Here are their most common denominators.
+
+<div class="alert alert-note" role="alert">
+Quick summary:
+
+<ul>
+  <li>Always test on the latest version.</li>
+  <li>Always provide step-by-step reproduction instructions.</li>
+  <li>Always give details, such as server configuration</li>
+  <li>Try to add pictures/video.</li>
+  <li>Try to enable the Debug Mode.</li>
+  <li>Try to check if the issue has not already been reported.</li>
+  <li>Try to add log files.</li>
+</ul>
+</div>
+
+
+### The basics
+
+Apart from the obvious descriptive title, there is a trifecta of must-haves in a useful Forge ticket.
+
+#### Up-to-date
+
+Oftentimes, bug reporters provide us with information about what happens in their own situation, chiefly their current version of PrestaShop -- which is seldom the latest version.
+
+This can make reviewing harder, because there is a number of things that could have changed between the reporter's version and the latest version of PrestaShop -- even fixing that very bug!
+
+That is why it is always preferable for the reporter to make sure that the bug happens in the latest available version -- either [an official release](https://www.prestashop.com/en/download), or [the current in-development version](https://github.com/PrestaShop/PrestaShop/tree/develop).
+
+Because not everyone can afford the time to upgrade their online shop to the latest version in hope that their issue is fixed, we encourage the more technical users to make tests on their local machines, using an Apache-PHP-MySQL/MariaDB stack such as [WampServer](http://www.wampserver.com/), [MAMP](https://www.mamp.info/en/), [XAMPP](https://www.apachefriends.org/) or any other.<br/>
+Not only will it then be able to easily install the latest version, but it will also make it possible to test in the default conditions -- indeed, third-party themes and modules can create issues that the PrestaShop simply cannot fix.
+
+
+#### Reproducible
+
+Writing how to reproduce an issue every single time helps tremendously. Even better if its reproducible in fresh install of PrestaShop.
```

Comment:
Writing how to reproduce an issue time helps tremendously. Even better if its reproducible in a fresh install of PrestaShop.